### PR TITLE
add new products openeuler and openeuler2203 supporting

### PR DESCRIPTION
**Pull request by StevenYGui**
#### Description:

- OpenScap is a well-known security configuration baseline scanning organization in the industry. The tools provided by OpenScap provide security scanning functions for most operating systems around the world. OpenEuler also hopes to join the OpenScap family and use the powerful functions of OpenScap to complete and provide security configuration baseline scanning capabilities.

- The openEuler community, known as the OpenAtom openEuler community, is an open source community for operating systems. The openEuler or openEuler community is incubated and operated by the Open Atomic Open Source Foundation.

- The openEuler operating system supports application scenarios such as servers, cloud computing, edge computing, and embedded computing, and supports diversified computing. It is committed to providing a secure, stable, and easy-to-use operating system. Support OT applications and the integration of OT and ICT by providing qualitative assurance capabilities for applications.

- Through an open community, the openEuler community works with global developers to build an open, pluralistic and inclusive software ecosystem. Incubate and support a variety of processor architectures, cover the whole scene of the word infrastructure, and promote the development of the software and hardware of the enterprise's word infrastructure and the application ecology.

- As of June 2023, the number of openEuler community members exceeded 970 and the number of community downloads exceeded 1.45 million. With more than 14,000 developers, its living global contributors are likely to continue to promote the global ecology of openEuler.

- openeuler2203 is a long term support version of openEuler.

#### Rationale:

- This pr contains two code submissions. One is to build the openEuler-related support framework, and the other is to add common openEuler check items.

#### Review Hints:

- Please review it and any feedback is my appreciate.

### CODE REVIEW

File: controls/std_openeuler.yml [hunk 1]
```diff
@@ -0,0 +1,1812 @@
+---
+policy: 'Standard Benchmark for openEuler'
```
Comment (vojtapolasek) [inline controls/std_openeuler.yml:2]: Does the policy have some resource? E.G. URL to the original document?

File: controls/std_openeuler.yml [hunk 2]
```diff
@@ -0,0 +1,1812 @@
...
+title: 'Standard Benchmark for openEuler'
+id: std_openeuler
+version: '1.0'
+levels:
+  - id: base
+
+controls:
```
Comment (vojtapolasek) [inline controls/std_openeuler.yml:9]: Not sure if it is a good idea to create reference IDs containing words as well... Are they named like that in the original security policy? I think numbers could be enough, you can put the description in the "title" tag.

**Review by vojtapolasek — CHANGES_REQUESTED** [review]
Hello and thank you for this contribution. I also apologize for such a long delay.
Please see my particular comments.
I have also two general questions:
1. I see that you submit two products: openeuler and openeuler2203. I understand that the openeuler2203 is an LTS version... but what about the openeuler? Is it not versioned? Is it a rolling distribution?
2. Could you please also modify the build_product script so that products added by this PR can be built with this script?
3. Thank you.

**Comment by StevenYGui** [comment]
> Hello and thank you for this contribution. I also apologize for such a long delay. Please see my particular comments. I have also two general questions:
> 
> 1. I see that you submit two products: openeuler and openeuler2203. I understand that the openeuler2203 is an LTS version... but what about the openeuler? Is it not versioned? Is it a rolling distribution?
> 2. Could you please also modify the build_product script so that products added by this PR can be built with this script?
> 3. Thank you.

very thanks for your response, I also applogize for the delay, for your questions, my answer as below:
1. yes, openeuler2203 is an LTS version, openeuler is a general version, just like 2009/2209 and so on, also including 2203 LTS version, acturally, it means that all openEuler versions can be checked without distinguishing specific version numbers. So, is it ok? If not, I think I can remove openeuler and keep openeuler2203.
2. ok, I will study it and try it.

Thank you very mush.

File: controls/std_openeuler.yml (hunk 1 above)
Comment (StevenYGui) [reply controls/std_openeuler.yml:2]: yes, i have, i will paste it latter

File: controls/std_openeuler.yml (hunk 2 above)
Comment (StevenYGui) [reply controls/std_openeuler.yml:9]: I'm sorry, I don't quite understand your question, but you're saying it's not appropriate to use the word "base" in the id field, I can use a number, or something like "level 1", is it right?

**Comment by StevenYGui** [comment]
it is closed, and I created a new PR: https://github.com/ComplianceAsCode/content/pull/11420

**Pull request closed without merging**